Update CPS placeholder pricing (#7552)
elastic/docs-content: deploy-manage/_snippets/cps-billing.md

During technical preview, there are no separate {{cps}} charges.

When {{cps-init}} becomes generally available, charges are billed to the origin project. 

Usage is metered and charged based on the following factors:

* **Retained data volume:** The volume of data retained in your linked projects each month. The rate varies by the type of each linked project, such as Observability, Security, or {{es}}. For Observability linked projects, the rate also depends on the type of retained data.
* **{{cps-init}} queries:** Cross-project search queries incur a separate data egress charge for data transferred as a result of those queries. This appears as a separate line item on your bill.

> From August 1, 2026, the [Elastic Cloud Pricing](https://cloud.elastic.co/cloud-pricing-table) page shows detailed pricing for various {{cps}} scenarios.

Reviewer: > From August 1, 2026, the [Elastic Cloud Pricing](https://cloud.elastic.co/cloud-pricing-table) page shows detailed pricing for various {{cps}} scenarios. 

I need to double check the date

Reviewer: confirmed Aug 1st

During technical preview, there are no separate {{cps}} charges.

When {{cps-init}} becomes generally available, charges are billed to the origin project. 

Usage is metered and charged based on the following factors:

* **Retained data volume:** The volume of data retained in your linked projects each month. The rate varies by the type of each linked project, such as Observability, Security, or {{es}}. For Observability linked projects, the rate also depends on the type of retained data.
* **{{cps-init}} queries:** Cross-project search queries incur a separate data egress charge for data transferred as a result of those queries. This appears as a separate line item on your bill.

> Starting on August 1, 2026, before billing starts, your [usage page](https://cloud.elastic.co/usage) shows a billing preview with projected {{cps}} charges based on your current usage. No charges apply until billing begins.

Reviewer 2: There's instances of `starting` and `starts` and `begins` all in very close proximity. Could we simplify this a bit?

```suggestion
From August 1, 2026, your [usage page](https://cloud.elastic.co/usage) shows a billing preview with projected {{cps}} charges based on your current usage. No charges apply until billing begins.
```

(I assumed the preference of using present instead of future is deliberate)

> To estimate monthly spend based on your linked project configuration, use the [{{ecloud}} pricing calculator](https://cloud.elastic.co/pricing).

Author: ```suggestion
From August 1, 2026, the [Elastic Cloud Pricing](https://cloud.elastic.co/cloud-pricing-table) page shows detailed pricing for various {{cps}} scenarios.
```
